prow/jenkins/controller.go

```diff
@@ -99,13 +99,23 @@ func NewController(kc *kube.Client, jc *Client, ghc *github.Client, logger *logr
 	}
 }
 
+func (c *Controller) config() config.Controller {
+	for _, cfg := range c.ca.Config().JenkinsOperator {
+		if cfg.LabelSelector == c.selector {
+			return cfg.Controller
+		}
+	}
+	// Default to something if no config exists.
```

Comment:
This retains the default behavior. Not clear what actually happens when somebody runs the controller w/o having any config set up (w/ respect to the url templates).